### ISSUE TITLE
explain_prediction support for decision trees and ensemble 

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#11]
See https://github.com/andosa/treeinterpreter

**Comment by lopuhin** [comment]
Maybe ``desicion_path`` from sklearn 0.18 can help here: http://scikit-learn.org/stable/modules/generated/sklearn.tree.ExtraTreeClassifier.html#sklearn.tree.ExtraTreeClassifier.decision_path

**Comment by kmike** [comment]
Yeah, it can help for decision trees. 

More approaches:

* https://arxiv.org/abs/1606.05390 (code: https://github.com/sato9hara/defragTrees);
* https://github.com/IBCNServices/GENESIM

**Comment by lopuhin** [comment]
Related to #117.
Thanks for the links, the code from the first paper looks especially interesting!

**Comment by kmike** [comment]
I'm closing it as fixed by https://github.com/TeamHG-Memex/eli5/pull/122 and https://github.com/TeamHG-Memex/eli5/pull/117. We can investigate additional methods separately.
